pennylane_lightning/core/src/simulators/lightning_kokkos/gates/MatrixGateFunctors.hpp

```diff
@@ -539,4 +522,4 @@ template <class PrecisionT> struct apply5QubitOpFunctor {
     }
 };
 
-} // namespace Pennylane::LightningKokkos::Functors
+} // namespace Pennylane::LightningKokkos::Functors
```

Comment:
Could you double check these EOF rendering issues? There are easy to fix but will require some last checks...
```suggestion
} // namespace Pennylane::LightningKokkos::Functors

```

Reply:
I've updated my vscode setting now to add new final line on save! Thanks :)